build/repositories.bzl

```diff
@@ -51,12 +51,12 @@ def _copyright_header(ctx):
         if path.endswith(".js") or path.endswith(".map") or path.endswith(".css"):
             content = ctx.read(path)
             if not content.startswith(copyright_content_js):
-                ctx.file(path, copyright_content_js + content)
+                ctx.file(path, copyright_content_js + content, legacy_utf8 = False)
 
         elif path.endswith(".html"):
             content = ctx.read(path)
             if not content.startswith(copyright_content_html):
-                ctx.file(path, copyright_content_html + content)
+                ctx.file(path, copyright_content_html + content, legacy_utf8 = False)
```

Comment:
we might want to add a test for this bug, to ensure we are not incorrectly re-encoding the file.
we can do it in the explain_manifest script https://github.com/Kong/kong/blob/master/scripts/explain_manifest/suites.py#L2

Reply:
In this repo, we don't actually download anything, so there is nothing that needs to be tested.
But it's still very hard to detect the re-encoding in the EE version since we may not have enough knowledge of the source text. The corrupted text is still legal, just some string changes.

Reply:
Let's leave it as is then, we might still want to test it, probably outside of this repo, using frontend testing tools. as this is not a blocker for now, i'm resolving this.